### DISCUSSION TITLE
What's the difference between TabView and TabMenu?

### DISCUSSION THREAD

**Discussion by MAN-Sendance** [primefaces/community#1236] [PrimeNG] [answered]
Are TabView and TabMenu essentially the same, just configured differently? Are there distinct use cases for each, or do they serve similar purposes with slight variations in configuration?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

TabView is to manage view by changing tabs it doesn't support menu accessibility and routing functionalities. TabMenu is a menu component which used to create a menu for an app.
